.github/workflows/binaries.yml

```diff
@@ -0,0 +1,61 @@
+name: Generate binaries
+on:
+  release:
+    types:
+      - created
+jobs:
+  binaries:
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            asset_name: api-linux-amd64.tar.gz
+            compress_cmd: tar -czvf
+            build_cmd: docker run -v $PWD:/data golang:1.19 bash -c "cd /data && go build ./cmd/api"
```

Comment:
You might ask, why not: `go build ..` style?
That was the first approach, but unfortunately, the GLIBC version that Github Actions VMs are using is quite new, and I couldn't even run that binary on my Ubuntu desktop machine. So what I did here compile the binary inside the `golang:1.19` docker image that has a lower version of GLIBC that will be compatible with more machines.

This might be a good idea to be applied in the other binaries, but unless somebody reports this happening in other places  maybe we can avoid it.